autogpt_platform/backend/backend/blocks/__init__.py

```diff
@@ -20,7 +20,7 @@ def load_all_blocks() -> dict[str, type["Block"]]:
     modules = [
         str(f.relative_to(current_dir))[:-3].replace(os.path.sep, ".")
         for f in current_dir.rglob("*.py")
-        if f.is_file() and f.name != "__init__.py"
+        if f.is_file() and f.name != "__init__.py" and not f.name.startswith("test_")
```

Comment:
maybe also include a pattern for `*_test.py`?
```suggestion
        if (
            f.is_file()
            and f.name != "__init__.py"
            and not f.name.startswith("test_")
            and not f.name.endswith("_test.py")
        )
```